dotcom-rendering/src/web/components/OnwardsUpper.importable.tsx

```diff
@@ -313,3 +305,7 @@ export const OnwardsUpper = ({
 		</div>
 	);
 };
+
+export const _ = {
+	firstPopularTag,
+};
```

Comment:
I'm not quite sure I understand why we're exporting this as a part of `_`? Is this a new pattern for exporting only for testing purposes?

Reply:
I’ve seen it done before in `frontend`, but actually, I’m not a fan. It would be quite difficult to test the actual rendered object as there is some quite deeply nested logic in there.

I’ll use a simple export!